Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -233,7 +233,7 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
         
         context.restoreGState()
         
-        renderer!.drawExtras(context: context)
```

Comment:
Why not use
`guard let data = data, let renderer = renderer else { return }`
at the beginning of the function?

Logically, a `draw()` function would require a renderer to draw anything, and looking through the implementation of the function this is true.

Reply:
done